src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -264,6 +264,10 @@ private static Request ToRequest(FileConfig fileConfig, string path)
             Capture[] c = CFamilyHelper.Capture.ToCaptures(fileConfig, path, out string cfamilyLanguage);
             var request = MsvcDriver.ToRequest(c);
             request.CFamilyLanguage = cfamilyLanguage;
+            if (fileConfig.ItemType == "ClInclude")
+            {
+                request.Flags |= Request.MainFileIsHeader;
+            }
```

Comment:
I think it is better to move these 3 lines to inside `ToRequest` in `MsvcDriver.cs` where we set all the flags.